storage/plugins/archives/ArchivesPlugin.php

```
@@ -3983,6 +4019,792 @@ public function plannedHooks(): array
+            $xw->endElement(); // header
+
+            if (!$identifiersOnly) {
+                $xw->startElement('metadata');
+                $authorities = $this->fetchAuthoritiesForArchivalUnit($rowId);
+                if ($metadataPrefix === 'oai_dc') {
+                    $this->writeDublinCoreRecord($xw, $row, $authorities);
+                } elseif ($metadataPrefix === 'ead3') {
+                    $this->writeEad3Document($xw, $row, $authorities);
+                } else {
+                    $this->writeArchivalUnitMarcRecord($xw, $row, $authorities);
+                }
```

PHPMD (2.15.0)

[error] The method oaiListRecords uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)